### ISSUE TITLE
Turbostat is x86 only

### ISSUE THREAD

**Issue by hrw** [intel/lkp-tests#34]
Going through steps from [lkp-tests.pdf](https://01.org/sites/default/files/documentation/lkp-tests.pdf) I got stopped on 'turbostat':
```
Making turbostat benchmark for debian                  
timeout 60m git clone -q https://github.com/aaronlu/turbostat.git turbostat
gcc -Wall turbostat.c -c -o /tmp/build-turbostat/turbostat/turbostat.o
turbostat.c:43:10: fatal error: cpuid.h: No such file or directory
 #include <cpuid.h>                                   
          ^~~~~~~~~
compilation terminated.
make: *** [Makefile:14: turbostat.o] Error 1
Making turbostat-LKP failed
```

Looked at source and noticed that it is x86-64 only tool. And I am on AArch64...

Can turbostat be optional?

**Comment by zhijianli88** [comment]
> Looked at source and noticed that it is x86-64 only tool. And I am on AArch64..
you are right.

> 
> Can turbostat be optional?
you can disable it locally with following changes

```shell
lizhijian@lkp-boy:~/lkp-tests$ git diff
diff --git a/include/category/benchmark b/include/category/benchmark
index c196e7a8..1e57bf2d 100644
--- a/include/category/benchmark
+++ b/include/category/benchmark
@@ -21,7 +21,6 @@ diskstats:
 nfsstat:
 cpuidle:
 cpufreq-stats:
-turbostat:
 sched_debug:
 perf-stat:
 mpstat:
```

**Comment by zhijianli88** [comment]
this issue could be fixed by: 704abc37427d ("include/category/benchmark: export turbostat only if on arch x86/x86_64")
